content/en/docs/Examples/bandwidth-optimized-upgrades.md

```diff
@@ -0,0 +1,282 @@
+---
+title: "Bandwidth Optimized Upgrades"
+linkTitle: "Bandwidth Optimized Upgrades"
+description: This section describes how to optimize bandwidth usage during OS upgrades using distributed caching solutions.
+---
+
+{{% alert title="Info" color="info" %}}
+This tutorial demonstrates how to optimize bandwidth usage during OS upgrades using distributed caching solutions like embedded registries.
+{{% /alert %}}
+
+# Introduction
+
+Nodes in edge clusters often have poor networking capabilities, and Kairos users may create custom images that are significantly larger (e.g., by including many kernel drivers). The current issue is that during upgrades, each node in the cluster must re-download the entire image from scratch before applying the upgrade.
+
+This documentation explores solutions to optimize bandwidth usage during upgrades by implementing distributed caching mechanisms.
+
+# Problem Statement
+
+- **Large Images**: Custom images can be very large due to additional drivers, tools, or configurations
+- **Poor Network**: Edge nodes often have limited or unreliable network connectivity
+- **Redundant Downloads**: Each node downloads the same upgrade image independently
+- **Bandwidth Waste**: Multiple nodes downloading identical images consumes unnecessary bandwidth
+
+# Solutions
+
```

Comment:
maybe just a little statement mentioning that right now solutions focus on standard images?

Reply:
good point, added